phoenix-core/src/it/java/org/apache/phoenix/end2end/QueryDatabaseMetaDataIT.java

```diff
@@ -190,14 +190,8 @@ public void testSchemaMetadataScan() throws SQLException {
 
         rs = dbmd.getSchemas(null, null);
         assertTrue(rs.next());
-        assertEquals(rs.getString("TABLE_SCHEM"),null);
```

Comment:
As now , we will show only those schemas which are created with "create schema" command and having table as empty string. so null is not expected